api/src/test/java/org/openmrs/validator/PersonValidatorTest.java

```diff
@@ -166,8 +185,7 @@ public void validate_shouldFailValidationIfFieldLengthsAreNotCorrect() throws Ex
 		person.setBirthdate(new Date());
 		person.setGender("too long text too long too long text too long text  too long text");
 		person.setPersonVoided(true);
-		person
```

Comment:
Did you touch this? Or did you forget to run mvn clean install before committing as advised at https://wiki.openmrs.org/display/docs/Pull+Request+Tips